sql/sys_vars.cc

```diff
@@ -6683,3 +6683,28 @@ static Sys_var_ulonglong Sys_max_rowid_filter_size(
        SESSION_VAR(max_rowid_filter_size), CMD_LINE(REQUIRED_ARG),
        VALID_RANGE(1024, (ulonglong)~(intptr)0), DEFAULT(128*1024),
        BLOCK_SIZE(1));
+
+static Sys_var_mybool Sys_redirect_enabled(
+       "redirect_enabled",
+       "Enable return redirection endpoint",
+       GLOBAL_VAR(redirect_enabled), CMD_LINE(OPT_ARG),
+       DEFAULT(FALSE), NO_MUTEX_GUARD, NOT_IN_BINLOG, ON_CHECK(0),
+       ON_UPDATE(0));
+
+static Sys_var_charptr Sys_redirect_server_host(
+       "redirect_server_host",
+       "The server host that client can choose to redirect to.",
+       READ_ONLY GLOBAL_VAR(redirect_server_host), CMD_LINE(OPT_ARG),
+       DEFAULT("localhost"));
+
+static Sys_var_charptr Sys_redirect_server_port(
```

Comment:
are you keeping this a charptr so the url can include named ports (/etc/services)? Or is this just so you can use `strcat`?

Reply:
To be frank, I haven't tried named ports yet; but I guess defining port as a charptr does open up such possibility.